src/app/heroes/heroes.component.html

```diff
@@ -1,5 +1,7 @@
-<h2>{{hero.name | uppercase}} Details</h2>
-<div><span>id: </span>{{hero.id}}</div>
+<div *ngIf="selectedHero">
+  <h2>{{selectedHero.name | uppercase}} Details</h2>
+  <div><span>id: </span>{{selectedHero.id}}</div>
+</div>
```

Comment:
Ek dink al my changes hier is maar net mid-tutorial errors. Ek dink as jy die stukkie net weer volg in die tutorial sal jy by die regte kode uitkom 👍